zetaclient/chains/bitcoin/rpc/rpc.go

```diff
@@ -20,6 +23,10 @@ const (
 
 	// defaultTestnetFeeRate is the default fee rate for testnet, 10 sat/byte
 	defaultTestnetFeeRate = 10
+
+	// rpcLatencyThreshold is the threshold for RPC latency to be considered unhealthy
+	// Bitcoin block time is 10 minutes, 1200s (20 minutes) is a reasonable threshold for Bitcoin
+	rpcLatencyThreshold = 1200
```

Comment:
Couldn't be something as part of the config?

Reply:
added to config: https://github.com/zeta-chain/node/pull/2751/commits/fedd2fead3d17535cd748e8bac7cb97cbae906be